ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -463,7 +464,11 @@ class EclCmdPublish : public EclCmdWithEclTarget
         if (resp->getReloadFailed())
             fputs("\nAdded to Queryset, but request to reload queries on cluster failed\n", stderr);
 
-        return outputMultiExceptionsEx(resp->getExceptions());
+        int ret = outputMultiExceptionsEx(resp->getExceptions());
+        if (outputQueryFileCopyErrors(resp->getFileErrors()))
+            ret = 1;
```

Comment:
The return is an error code, not a counter.  1 means "general error".  If we wanted we could use very specific error codes, and warning codes.  0 is always OK, positive numbers are used as errors and negative numbers as warning.